deployments/liqo/templates/liqo-vk-options-template.yaml

```diff
@@ -0,0 +1,82 @@
+{{- $vkargs = append $vkargs "--enable-apiserver-support=true" }}
+{{- end }}
+{{- /* Configure the appropriate flags if the IP reflection is disabled, if not overridden by the user */ -}}
+{{- if not .Values.networking.reflectIPs }}
+{{- if not (or (has "--disable-ip-reflection" $vkargs ) (has "--disable-ip-reflection=true" $vkargs ) (has "--disable-ip-reflection=false" $vkargs )) }}
+{{- $vkargs = append $vkargs "--disable-ip-reflection" }}
+{{- end }}
+{{- end }}
+{{- /* Configure the appropriate certificate generation approach on EKS clusters, if not overridden by the user */ -}}
+{{- if .Values.authentication.awsConfig.accessKeyId }}
+{{- if not (or (has "--certificate-type=kubelet" $vkargs ) (has "--certificate-type=aws" $vkargs ) (has "--certificate-type=self-signed" $vkargs )) }}
+{{- $vkargs = append $vkargs "--certificate-type=aws" }}
+{{- end }}
+{{- end }}
+
+apiVersion: virtualkubelet.liqo.io/v1alpha1
+kind: VkOptionsTemplate
+metadata:
+  name: {{ printf "%s-default" $kubeletConfig.name }}
+  labels:
+    {{- include "liqo.labels" $kubeletConfig | nindent 4 }}
+spec:
+  createNode: {{ .Values.offloading.createNode }}
+  disableNetworkCheck: {{ .Values.offloading.disableNetworkCheck }}
+  containerImage: {{ .Values.virtualKubelet.image.name }}{{ include "liqo.suffix" $kubeletConfig }}:{{ include "liqo.version" $kubeletConfig }}
+  metricsEnabled: {{ .Values.virtualKubelet.metrics.enabled }}
+  {{- if .Values.virtualKubelet.metrics.enabled }}
+  metricsAddress: :{{ .Values.virtualKubelet.metrics.port }}
+  {{- end }}
+  {{- if ge (len .Values.offloading.reflection.skip.labels) 1 }}
+  labelsNotReflected:
+    {{- toYaml .Values.offloading.reflection.skip.labels | nindent 4 }}
+  {{- end }}
+  {{- if ge (len .Values.offloading.reflection.skip.annotations) 1 }}
+  annotationsNotReflected:
+    {{- toYaml .Values.offloading.reflection.skip.annotations | nindent 4 }}
+  {{- end }}
+  reflectorsWorkers:
+    pod: {{ .Values.offloading.reflection.pod.workers }}
+    service: {{ .Values.offloading.reflection.service.workers }}
+    endpointslice: {{ .Values.offloading.reflection.endpointslice.workers }}
+    ingress: {{ .Values.offloading.reflection.ingress.workers }}
+    configmap: {{ .Values.offloading.reflection.configmap.workers }}
+    secret: {{ .Values.offloading.reflection.secret.workers }}
+    serviceaccount: {{ .Values.offloading.reflection.serviceaccount.workers }}
+    persistentvolumeclaim: {{ .Values.offloading.reflection.persistentvolumeclaim.workers }}
+    event: {{ .Values.offloading.reflection.event.workers }}
+  reflectorsType:
+    service: {{ .Values.offloading.reflection.service.type }}
+    ingress: {{ .Values.offloading.reflection.ingress.type }}
+    configmap: {{ .Values.offloading.reflection.configmap.type }}
+    secret: {{ .Values.offloading.reflection.secret.type }}
+    event: {{ .Values.offloading.reflection.event.type }}
```

Comment:
I don't know if it's better to have this on this form or change the hierarchy

```
reflection:
  pod:
    workers:
    type:
  service:
    workers:
    type:
...
```

in order to reuse the same small object for each resource and make it easier to add new ones in the future